pkg/cmd/roachtest/roachtestutil/mixedversion/planner.go

```diff
@@ -1362,6 +1363,37 @@ func (ss stepSelector) Filter(predicate func(*singleStep) bool) stepSelector {
 	return result
 }
 
+// Cut returns the before and after of a cut step selector, as well as the step that was cut out.
+func (ss stepSelector) Cut(
+	predicate func(*singleStep) bool,
+) (stepSelector, stepSelector, stepSelector) {
```

Comment:
nit: you could name these return args: 

```
func (ss stepSelector) Cut(
	predicate func(*singleStep) bool,
) (before, after, cut stepSelector) {
	predicateFound := false
```

Makes its slightly nicer to figure out the return values.